data/sles4sap/agama/sles_sap_default.jsonnet

```diff
@@ -0,0 +1,22 @@
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE_SLES4SAP}}',
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true,
```

Comment:
No idea how this works but I see you have additional line ion ppc64le file 
```
sshPublicKey: 'enable ssh',
```
Is it not required here?

Reply:
I think it was not there as on `qemu` the SUTs run on the worker, but I'll add it anyway. Thanks.

Reply:
Done.